conda_smithy/templates/azure-pipelines-win.yml.tmpl

```diff
@@ -57,8 +57,8 @@ jobs:
     - task: CondaEnvironment@1
       inputs:
         packageSpecs: 'python=3.9 conda-build conda pip {{ BOA }}{{ " ".join(remote_ci_setup) }}' # Optional
-        installOptions: "-c conda-forge"
-        updateConda: true
+        installOptions: "-c conda-forge --override-channels"
+        updateConda: false
```

Comment:
we should be able to update conda with `--override-channels` now right?

Reply:
If the action would take that into account, then yes, but I do not think it does: https://github.com/microsoft/azure-pipelines-tasks/blob/2135c795837e3692569b3baa4cccefc9f4c9b94d/Tasks/CondaEnvironmentV1/conda_internal.ts#L93-L102

Note this other line: https://github.com/microsoft/azure-pipelines-tasks/blob/2135c795837e3692569b3baa4cccefc9f4c9b94d/Tasks/CondaEnvironmentV1/conda_internal.ts#L121-L123

Reply:
ok